karb/parser.js

```diff
@@ -3,13 +3,33 @@
 'use strict';
 const Parser = require('../.lib/parser.js');
 
+/**
+ * Extract rtype
+ */
+function determineRtype (param, result) {
+  let idMatch;
```

Comment:
Not necessary in this case, prefer write ``const idMatch = /^(.+)Ch[0-9]+$/.exec(ids);`` in line 12